requirements/requirements_asr.txt

```diff
@@ -5,13 +5,15 @@ ipywidgets
 jiwer
 kaldi-python-io
 kaldiio
+lhotse>=1.19.2
```

Comment:
If i remember correctly, lhotse doesnt have compiled binary right? So there's no real need to even import guard, you can just put the imports at the top of the file?

Reply:
Yeah, no compiled binaries required. I will just remove the local imports / import guards. These are left-overs from an initial design where Lhotse was an optional dependency.